a.el

```diff
@@ -143,10 +147,15 @@
                           that))
            colls))
 
+;; TODO a-merge-with 
+
 (defun a-alist (&rest kvs)
   "Create an association list from the given keys and values, provided as a single list of arguments. e.g. (a-alist :foo 123 :bar 456)"
   (mapcar (lambda (kv) (cons (car kv) (cadr kv))) (-partition 2 kvs)))
 
+;; XXX shouldn't this fn use the top of the top level collection to
+;; create children? that is, shouldn't (a-assoc-in (make-hash-table)
+;; ...) produce nested hash tables?
```

Comment:
Not a terribly strong opinion on this one. I can see the reasoning, but I wouldn't say that's particularly intuitive behavior either. I'm fine with this function (and the lib in general) leaning a bit towards preferring alists.